components/layout_2020/style_ext.rs

```diff
@@ -535,6 +578,77 @@ impl ComputedValuesExt for ComputedValues {
         false
     }
 
+    /// Resolve the preferred aspect ratio according to the given natural aspect
+    /// ratio and the `aspect-ratio` property.
+    /// https://drafts.csswg.org/css-sizing-4/#aspect-ratio
```

Comment:
Raw URLS in rustdoc comments lead to rustdoc warnings:

```suggestion
    /// <https://drafts.csswg.org/css-sizing-4/#aspect-ratio>
```